spark/src/main/java/io/delta/dynamodbcommitstore/integration_tests/dynamodb_commitowner_integration_test.py

```diff
@@ -0,0 +1,233 @@
+num_rows = int(os.environ.get("DELTA_NUM_ROWS", 16))
+dynamo_endpoint = os.environ.get("DELTA_DYNAMO_ENDPOINT", "https://dynamodb.us-west-2.amazonaws.com")
+
+# ===== Optional input from user (we calculate defaults using RUN_ID) =====
+relative_delta_table_path = os.environ.get("RELATIVE_DELTA_TABLE_PATH", f"tables/table_ddb_cs_{run_id}_{str(uuid.uuid4())}")\
+    .rstrip("/")
+dynamo_table_name = os.environ.get("DELTA_DYNAMO_TABLE_NAME", "test_ddb_cs_table_" + run_id)
+
+delta_table_path = "s3a://" + s3_bucket + "/" + relative_delta_table_path
+relative_delta_log_path = relative_delta_table_path + "/_delta_log/"
+
+if delta_table_path is None:
+    print(f"\nSkipping Python test {os.path.basename(__file__)} due to the missing env variable "
+          f"`DELTA_TABLE_PATH`\n=====================")
+    sys.exit(0)
+
+dynamodb_commit_owner_conf = json.dumps({
+    "managedCommitsTableName": dynamo_table_name,
+    "dynamoDBEndpoint": dynamo_endpoint
+})
+
+test_log = f"""
+==========================================
+run id: {run_id}
+delta table path: {delta_table_path}
+dynamo table name: {dynamo_table_name}
+
+concurrent writers: {concurrent_writers}
+concurrent readers: {concurrent_readers}
+number of rows: {num_rows}
+
+relative_delta_table_path: {relative_delta_table_path}
+relative_delta_log_path: {relative_delta_log_path}
+==========================================
+"""
+print(test_log)
+
+commit_owner_property_key = "managedCommits.commitOwner"
+property_key_suffix = "-preview"
+
+spark = SparkSession \
+    .builder \
+    .appName("utilities") \
+    .master("local[*]") \
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
+    .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
+    .config(f"spark.databricks.delta.properties.defaults.{commit_owner_property_key}{property_key_suffix}", "dynamodb") \
+    .config(f"spark.databricks.delta.properties.defaults.managedCommits.commitOwnerConf{property_key_suffix}", dynamodb_commit_owner_conf) \
+    .config(f"spark.databricks.delta.managedCommits.commitOwner.ddb.awsCredentialsProviderName", "com.amazonaws.auth.profile.ProfileCredentialsProvider") \
+    .getOrCreate()
+
+# spark.sparkContext.setLogLevel("INFO")
+
+print("Creating table at path ", delta_table_path)
+spark.sql(f"CREATE table delta.`{delta_table_path}` (id int, a int) USING DELTA") # commit 0
+
+
+def write_tx(n):
+    print("writing:", [n, n])
+    spark.sql(f"INSERT INTO delta.`{delta_table_path}` VALUES ({n}, {n})")
+
+
+stop_reading = threading.Event()
+
+
+def read_data():
+    while not stop_reading.is_set():
+        print("Reading {:d} rows ...".format(
+            spark.read.format("delta").load(delta_table_path).distinct().count())
+        )
+        time.sleep(1)
+
+
+def start_read_thread():
+    thread = threading.Thread(target=read_data)
+    thread.start()
+    return thread
```

Comment:
Seems to be quite common utils in the other test as well, maybe should we create a dynamo_test_util.py and let both of them look at the files?